dair_pll/drake_utils.py

```diff
@@ -56,9 +56,8 @@
 CAM_FOV = np.pi/5
 VIDEO_PIXELS = [480, 640]
 FPS = 30
-SENSOR_POSE = RigidTransform(RollPitchYaw([-np.pi/2, 0, np.pi/2]), [1, 0, 0.2])
-EXP_NAME = file_utils.EXP_NAME
-VIDEO_FILENAME = file_utils.VIDEO_FILENAME
+SENSOR_POSE = RigidTransform(RollPitchYaw([-np.pi/2, 0, np.pi/2]), [2, 0, 0.2])
+VIDEO_FILENAME = file_utils.get_experiment_video_filename()
```

Comment:
Stylistically, probably not much of a reason to create a static variable that's just a single call to a member of `file_utils`